spec/queries/matched_for_notification_spec.rb

```diff
@@ -26,122 +26,124 @@ def execute_query(query_hash, field: :tags)
       end
 
-      it "finds subscriber lists matching all topics" do
-        lists = execute_query({ topics: %w[vat tax] })
-        expect(lists).to eq([@all_topics_tax_vat])
+      it "finds subscriber lists matching all policies" do
+        lists = execute_query({ policies: %w[vat tax] })
+        expect(lists).to eq([@all_policies_tax_vat])
       end
     end
 
-    context "matches on any and all topics" do
+    context "matches on any and all policies" do
       before do
-        @all_topics_tax_vat_any_topics_licensing_paye = create_subscriber_list_with_tags_facets(topics: { all: %w[vat tax], any: %w[licensing paye] })
-        @all_topics_tax_vat_licensing_any_topics_paye = create_subscriber_list_with_tags_facets(topics: { all: %w[vat tax schools], any: %w[paye] })
+        @all_policies_tax_vat_any_policies_licensing_paye = create_subscriber_list_with_tags_facets(policies: { all: %w[vat tax], any: %w[licensing paye] })
+        @all_policies_tax_vat_licensing_any_policies_paye = create_subscriber_list_with_tags_facets(policies: { all: %w[vat tax schools], any: %w[paye] })
       end
 
-      it "finds subscriber lists matching on both of all and one of any topics" do
-        lists = execute_query({ topics: %w[vat tax licensing] })
-        expect(lists).to eq([@all_topics_tax_vat_any_topics_licensing_paye])
+      it "finds subscriber lists matching on both of all and one of any policies" do
+        lists = execute_query({ policies: %w[vat tax licensing] })
+        expect(lists).to eq([@all_policies_tax_vat_any_policies_licensing_paye])
       end
 
-      it "does not find subscriber list for mixture of all and any topics when not all topics present" do
-        lists = execute_query({ field: :links, query_hash: { topics: %w[vat licensing] } })
-        expect(lists).not_to include(@all_topics_tax_vat_any_topics_licensing_paye)
+      it "does not find subscriber list for mixture of all and any policies when not all policies present" do
+        lists = execute_query({ field: :links, query_hash: { policies: %w[vat licensing] } })
+        expect(lists).not_to include(@all_policies_tax_vat_any_policies_licensing_paye)
       end
     end
 
-    context "matches on all topics and any policies" do
+    context "matches on all browse pages and any policies" do
       before do
-        @all_topics_tax_vat_any_policies_economy_industry = create_subscriber_list_with_tags_facets(topics: { all: %w[vat tax] }, policies: { any: %w[economy industry] })
-        @all_topics_vat_any_policies_economy_industry = create_subscriber_list_with_tags_facets(topics: { all: %w[paye schools] }, policies: { any: %w[economy industry] })
+        stub_const("ValidTags::ALLOWED_TAGS", ValidTags::ALLOWED_TAGS + %i[browse_pages])
+        @all_browse_pages_tax_vat_any_policies_economy_industry = create_subscriber_list_with_tags_facets(browse_pages: { all: %w[vat tax] }, policies: { any: %w[economy industry] })
```

Comment:
As above, browse_pages aren't a valid tag, so we should replace with taxons

Reply:
There are more incidences below, but I won't comment on each one

Reply:
Please, see the comment above.

This was done this way, because the only valid tags now are browse_pages and policies, but browse_pages are not in ValidTags::ALLOWED_TAGS. Some tests require two different tags to work, but after removing topics we are left with only one tag which is also in ValidTags::ALLOWED_TAGS, namely policies. So by adding browse_pages it allowed for two different tags, so the tests can work again.

Reply:
I'll use some of the specialist-publisher tags instead.